docs/development/test-vectors.rst

```diff
@@ -546,6 +546,15 @@ Custom X.509 Vectors
   This is an invalid certificate per :rfc:`5280` 4.2.1.12.
 * ``malformed-san.pem`` - A certificate with a malformed SAN.
 * ``malformed-ian.pem`` - A certificate with a malformed IAN.
+* ``admissions_extension_cert_e256.pem`` - A certificate with the ``Admissions``
```

Comment:
Can you do a separate PR with these? We like to review them separately from the implementation.

Reply:
@alex done! But unfortunately I had to extend the test and add an additional certificate in order to cover all corner cases - I hope this is fine.